content/docs/command-reference/stage/list.md

```diff
@@ -10,55 +10,57 @@ usage: dvc stage list [-h] [-q | -v]
                       [targets ...]
 
 positional arguments:
-  targets          Show stages from a dvc.yaml file or a directory (with -R).
-                   'dvc.yaml' by default.
+  targets       Limit command scope to these dvc.yaml files,
+                directories (with -R), or stage names.
+                './dvc.yaml' by default.
```

Comment:
Not 100% sure about this, using the "Limit command scope..." for consistency but could be different too. Or I can make a separate PR after this to get rid of that phrase everywhere.

- [ ] Will need core update

Reply:
Looks fine to me